portal/views/auth.py

```diff
@@ -521,14 +523,15 @@ def client():
     db.session.add(client)
     db.session.commit()
     auditable_event("added intervention/client {}".format(
-        client), user_id=user.id)
+        client), user_id=user.id, subject_id=client.user_id)
```

Comment:
i'd suggest `subject_id=user.id` - the subject here is really the "client" - and the client's user.id isn't unique - any user can have any number of clients, and to tie this change to the client.user_id is somewhat misleading.  when you add `context` it'll help.  i don't think it's worth the complexity of making subject a dynamic object type.  (i.e. subject might be a user, or a client, or a ...)

Same for rest of calls in this file.